### DISCUSSION TITLE
In which cases create user API may return 404 error

### DISCUSSION THREAD

**Discussion by ashep** [zitadel/zitadel#8550] [Q&A] [answered]
Hello!

I am using the [create user API](https://zitadel.com/docs/apis/resources/user_service_v2/user-service-add-human-user) and getting a 404 error in response. 

```
ID: QUERY-Dfbg2
Type: type.googleapis.com/zitadel.v1.ErrorDetail
Message: User could not be found
```

Could you please point out possible reasons for getting such an error?

Thank you!

**Comment by hifabienne** [comment]
can you send a curl example of the request you are sending?

**Reply by ashep** [reply]
It's pretty complex to reconstruct the entire environment I have using cURL requests. The thing is that the same API call works well on another Zitadel setup, and I have no idea what's the difference. So, I think the problem is not the request itself, but a particular Zitadel setup.

So, I wanted to ask what exactly can cause the API I've mentioned to return 404. I see that this error is possible according to the [documentation](https://zitadel.com/docs/apis/resources/user_service_v2/user-service-add-human-user), but there is no description of what cases a caller can get this error.

**Reply by ashep** [reply, marked as answer]
I got it. It was another API call error that was interfering with the create user's one. My bad. I apologize for mistakenly bothering you, @hifabienne.

**Reply by hifabienne** [reply]
Thanks for the clarification, great to hear that you got it working
